Gemfile

```diff
@@ -9,6 +9,7 @@ source "https://rubygems.org"
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
 gem "jekyll", "~> 3.7.2"
+gem 'jekyll-multiple-languages-plugin'
```

Comment:
We can specify the version, no?
And we can use also the `"` as in the previous line.

```
gem "jekyll-multiple-languages-plugin", "~> 1.6"
```

Reply:
Nice one!